core/server/services/webhooks/listen.js

```diff
@@ -34,24 +34,26 @@ function generatePayload(event, model) {
 }
 
 function listener(event, model, options) {
-    var payload = generatePayload(event, model);
+    let payload = {};
+    if (model) {
+        payload = generatePayload(event, model);
+    }
 
     // avoid triggering webhooks when importing
     if (options && options.importing) {
         return;
     }
 
-    api.webhooks.trigger(event, payload, options);
+    webhooks.trigger(event, payload, options);
 }
 
 // TODO: use a wildcard with the new event emitter or use the webhooks API to
 // register listeners only for events that have webhooks
 function listen() {
     common.events.on('subscriber.added', _.partial(listener, 'subscriber.added'));
     common.events.on('subscriber.deleted', _.partial(listener, 'subscriber.deleted'));
+    common.events.on('site.changed', _.partial(listener, 'site.changed'));
```

Comment:
Was the name of the event discussed?

Reply:
Asking: because it will show up in the client.

Reply:
Yes it was discussed. I imagine we'll probably have a map at some point with humanised labels once we have an `/events/` endpoint but for now the human names will be mapped and hardcoded in Ghost-Admin.

Reply:
👍